CRITICAL - 2021-03-15 11:14:02 --> Undefined index: email_login
#0 C:\xampp\htdocs\encurtador\app\Controllers\Login.php(38): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 38, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->signin()
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-03-15 11:14:08 --> Undefined index: email_login
#0 C:\xampp\htdocs\encurtador\app\Controllers\Login.php(38): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 38, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->signin()
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-03-15 11:29:09 --> mysqli_sql_exception: Unknown column 'usuarios.id' in 'field list' in C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 11:29:09 --> Unknown column 'usuarios.id' in 'field list'
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 15:11:43 --> Undefined index: url
#0 C:\xampp\htdocs\encurtador\app\Controllers\Link.php(68): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 68, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Link->store()
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Link))
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
ERROR - 2021-03-15 16:11:29 --> mysqli_sql_exception: Unknown column 'usuarios.id' in 'field list' in C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:11:29 --> Unknown column 'usuarios.id' in 'field list'
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-03-15 16:11:38 --> mysqli_sql_exception: Unknown column 'usuarios.id' in 'field list' in C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:11:38 --> Unknown column 'usuarios.id' in 'field list'
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
ERROR - 2021-03-15 16:11:39 --> mysqli_sql_exception: Unknown column 'usuarios.id' in 'field list' in C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:11:39 --> Unknown column 'usuarios.id' in 'field list'
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:20:36 --> Undefined variable: errors
#0 C:\xampp\htdocs\encurtador\app\Views\changepass.php(6): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 6, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('changepass', Array, true)
#4 C:\xampp\htdocs\encurtador\app\Controllers\User.php(65): view('changepass')
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->changePass()
#6 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
ERROR - 2021-03-15 16:24:09 --> mysqli_sql_exception: Unknown column 'usuarios.id' in 'field list' in C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:24:09 --> Unknown column 'usuarios.id' in 'field list'
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(314): mysqli->query('SELECT *, `link...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *, `link...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *, `link...')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1866): CodeIgniter\Database\BaseConnection->query('SELECT *, `link...', Array, false)
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Model.php(213): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\BaseModel.php(583): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\encurtador\app\Controllers\User.php(53): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:25:22 --> Undefined index: id_link
#0 C:\xampp\htdocs\encurtador\app\Views\meus_links.php(38): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 38, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('meus_links', Array, true)
#4 C:\xampp\htdocs\encurtador\app\Controllers\User.php(33): view('meus_links', Array)
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->index()
#6 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2021-03-15 16:42:38 --> Array to string conversion
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Common.php(1136): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1136, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Helpers\form_helper.php(59): stringify_attributes(Array)
#2 C:\xampp\htdocs\encurtador\app\Views\home.php(11): form_open('http://localhos...', Array)
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(220): include('C:\\xampp\\htdocs...')
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('home', Array, true)
#6 C:\xampp\htdocs\encurtador\app\Controllers\Home.php(12): view('home')
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Home->index()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2021-03-15 16:49:33 --> Arquivo inválido: esqueciSenha.php
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\View\View.php(203): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('esqueciSenha.ph...')
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Common.php(1217): CodeIgniter\View\View->render('esqueciSenha', Array, true)
#2 C:\xampp\htdocs\encurtador\app\Controllers\Login.php(65): view('esqueciSenha')
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\Login->esqueciSenha()
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#8 {main}
CRITICAL - 2021-03-15 17:08:36 --> Too few arguments to function App\Controllers\User::checkSenha(), 0 passed in C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php on line 936 and exactly 1 expected
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->checkSenha()
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-03-15 17:09:04 --> Undefined variable: email
#0 C:\xampp\htdocs\encurtador\app\Controllers\User.php(97): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 97, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->checkSenha()
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#3 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#6 {main}
CRITICAL - 2021-03-15 17:10:29 --> Call to undefined function App\Controllers\random_string()
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->checkSenha()
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-03-15 17:10:42 --> Call to a member function setTo() on string
#0 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->checkSenha()
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#5 {main}
CRITICAL - 2021-03-15 17:11:01 --> strpos() expects parameter 1 to be string, object given
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'strpos() expect...', 'C:\\xampp\\htdocs...', 790, Array)
#1 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Email\Email.php(790): strpos(Object(CodeIgniter\Email\Email), ',')
#2 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Email\Email.php(568): CodeIgniter\Email\Email->stringToArray(Object(CodeIgniter\Email\Email))
#3 C:\xampp\htdocs\encurtador\app\Controllers\User.php(112): CodeIgniter\Email\Email->setTo(Object(CodeIgniter\Email\Email))
#4 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(936): App\Controllers\User->checkSenha()
#5 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#6 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\encurtador\public\index.php(44): CodeIgniter\CodeIgniter->run()
#8 C:\xampp\htdocs\encurtador\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(45): require_once('C:\\xampp\\htdocs...')
#9 {main}
